qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -416,19 +425,32 @@ test_status_in_pool_dir()
 create_image()
 {
     local cluster=$1
-    local image=$2
+    local pool=$2
+    local image=$3
 
-    rbd --cluster ${cluster} -p ${POOL} create --size 128 \
+    rbd --cluster ${cluster} -p ${pool} create --size 128 \
```

Comment:
I think we need to fix all other functions too that use the hardcoded ${POOL}? test_image_replay_state, wait_for_replay_..., test_status_in_pool_dir...